app/models/comment.rb

```diff
@@ -3,13 +3,16 @@ class Comment < ApplicationRecord
   belongs_to :post
 
   after_save :update_comments_counter
+  after_destroy { update_pcomments_counter(false) }
 
```

Comment:
- [ ] Kindly take note that I was unable to delete comment as an `Admin` and `Authorised User`. I got the error message screenshot below 👇 

<img width="905" alt="Okota flyer-modified (3)" src="https://github.com/Timbar09/Blog_App/assets/103386183/870e80a0-c904-4526-8c45-fe752ca773a2">

Reply:
Thanks for the feedback, @OyePriscilla. I see the error and I'll fix it shortly.

Reply:
Fixed.